backstage/backstage
# [TechDocs Addons] give feedback addon

## Hey, I just made a Pull Request!

The PR moves the GiveFeedback addon (implemented in https://github.com/backstage/backstage/pull/10466) to live in the **techdocs plugin.** 

The reason to why its provided by the plugin now is that the techdocs addon framework has been moved to its own lightweight package in /packages.

## How to test

1. import the addon in the App.tsx
```tsx
import {GiveFeedbackAddon} from '@backstage/plugin-techdocs'
```
2. "activate" addon
```tsx
 <Route
      path="/docs/:namespace/:kind/:name/*"
      element={<TechDocsReaderPage />}
    >
      {techDocsPage}
      <TechDocsAddons>
        <GiveFeedbackAddon /> 
      </TechDocsAddons>
    </Route>
```

3. Start backstage locally by running `yarn dev` in the root. 
4. Navigate to a docs page
5. When highlighting a piece of text, the GiveFeedback addon should show up

<img width="1728" alt="Screenshot 2022-04-04 at 14 15 20" src="https://user-images.githubusercontent.com/25153114/161696009-ac577df8-cfc2-49e3-b301-dc4388945ed5.png">


#### :heavy_check_mark: Checklist
